presto-main/src/main/java/com/facebook/presto/operator/TableFinishOperator.java

```diff
@@ -79,17 +94,24 @@ public OperatorFactory duplicate()
 
     private final OperatorContext operatorContext;
     private final TableFinisher tableFinisher;
+    private final Operator statisticsAggregation;
```

Comment:
statisticsAggregationOperator?

Reply:
Didn't want to make it oververbose, but it looks like it indeed decreases readablity.

Reply:
Renamed `statisticsAggregationOperatorFactory` as well